src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -265,6 +280,14 @@ public static void openBrowser(String url) throws IOException {
         openExternalFilePlatformIndependent(fileType, url);
     }
 
+    public static void openBrowserCatchException(String url) {
+        try {
+            openBrowser(url);
+        } catch (IOException exception) {
+            LOGGER.error("Error while opening following ling with a browser: "+ url, exception);
+        }
+    }
+
```

Comment:
This is unnecessary as you can use the `openBrowser(String url)` method. This way you can react with a more specific error message or further processing.